<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChinhSua_Them</class>
 <widget class="QMainWindow" name="ChinhSua_Them">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>447</width>
    <height>499</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>447</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form
{
	background-image: url(:/background/BACKGROUND/bg1.jpg);
}</string>
  </property>
  <widget class="QWidget" name="Form">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>77</y>
      <width>391</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
	background-color: #222;
	border-radius: 20px
}

QPushButton
{
	background: red;
	border-radius: 10px;
}

QLineEdit
{
	background: transparent;
	border:none;
	color:#717072;
	border-bottom: 1px solid #717072;
}

QLabel
{
	color: white;
	font: 900 18pt &quot;Segoe UI&quot;;
}

QGroupBox
{
	border: 2px solid rgb(255, 255, 127);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>371</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="PRODUCT_ID_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Product ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRODUCT_ID_LINE">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>19</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Product ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ADD_PRODUCT_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>221</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ADD PRODUCT</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="EXIT_BUTTON">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>281</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 85, 127);</string>
      </property>
      <property name="text">
       <string>GO BACK</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="PRODUCT_NAME_LABEL">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>124</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRODUCT_NAME_LINE">
      <property name="geometry">
       <rect>
        <x>151</x>
        <y>120</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Product Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRODUCT_QUANTITY_LINE">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Product Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="PRODUCT_QUANTITY_LABEL">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>73</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quantity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="PRODUCT_PRICE_LABEL">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>179</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="PRODUCT_PRICE_LINE">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>Enter Product Price</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>127</x>
       <y>18</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 20pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>ADD PRODUCT</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>20</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	background-color: qconicalgradient(cx:0.489, cy:0.5, angle:0, stop:0.460227 rgba(246, 230, 116, 255));
	border-radius: 60px;
}
</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="ing.qrc">
      <normaloff>:/icon/ICON/add-to-basket.png</normaloff>:/icon/ICON/add-to-basket.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ing.qrc"/>
 </resources>
 <connections/>
</ui>
